Client/Frontend/Browser/BraveWebView.swift

```diff
@@ -46,4 +46,35 @@ class BraveWebView: WKWebView {
         lastHitPoint = point
         return super.hitTest(point, with: event)
     }
+
+    private func generateJavascriptFunctionString(functionName: String, args: [Any]) -> String {
+        var javascript = "\(functionName)("
+        for arg in args {
+            javascript += "'\(String(describing: arg).escapeHTML())', "
+        }
+
+        // Drop , and " "
+        javascript = javascript.dropLast(2) + ")"
+        return javascript
+    }
+
+    public func evaluateSafeJavascript(functionName: String, args: [Any], completion: @escaping ((Any?, Error?) -> Void)) {
+        let javascript = generateJavascriptFunctionString(functionName: functionName, args: args)
+        self.evaluateJavaScript(javascript) { data, error  in
+            completion(data, error)
+        }
+    }
+}
+
+extension String {
+    /// Encode HTMLStrings
+    func escapeHTML() -> String {
+       var result = self.replacingOccurrences(of: "&", with: "&amp;", options: .literal, range: nil)
+        result = result.replacingOccurrences(of: "\"", with: "&quot;", options: .literal, range: nil)
+        result = result.replacingOccurrences(of: "'", with: "&#39;", options: .literal, range: nil)
+        result = result.replacingOccurrences(of: "<", with: "&lt;", options: .literal, range: nil)
+        result = result.replacingOccurrences(of: ">", with: "&gt;", options: .literal, range: nil)
+        result = result.replacingOccurrences(of: "`", with: "&lsquo;", options: .literal, range: nil)
```

Comment:
`range: nil` on each of these lines can be removed.

You can also daisy chain all of these such as

```
return self
  .replacingOccurrences(of: "&", with: "&amp;", options: .literal)
  .replacingOccurrences(of: "\"", with: "&quot;", options: .literal)
  ...
```

But its not _really_ needed, just may look a bit nicer